js/explore/largeRepos/sunburst_licenses.js

```diff
@@ -0,0 +1,331 @@
+                                        y0: Math.max(0, d.y0 - o.depth),
+                                        y1: Math.max(0, d.y1 - o.depth)
+                                    });
+                                    return arc(d.current);
+                                }
+                            });
+                    return update;
+                }, exit => exit);
+                label.data(root.descendants().slice(1))
+                    .join('text')
+                        .attr('dy', '0.35em')
+                        .attr('fill-opacity', d => +labelVisible(d.current))
+                        .attr('transform', d => labelTransform(d.current))
+                        .text(d => {
+                            let name = d.data.name;
+                            if (d.height > 0) {
+                                return name;
+                            } else {
+                                name = name.split('/')[1];
+                                if(name.length > 20) {
+                                    return '...';
+                                } else {
+                                    return name;
+                                }
+                            }
+                        });
+                path.selectAll('title').remove();
+                path.append('title').text(d => d.data.name);
+            });
+    
+        // Creates option slider
+        /*chart.append('g')
+            .attr('transform', `translate(${margin.left},${margin.top / 2})`)
+            .attr('id', 'licenseSlider')
+            .call(slider);*/
+
+    }
+
+    // Turns json into useable data. Output MUST use arrays in accordance with the d3.hierarchy requirements
+    function reformatData(obj, type) {
+        // Converts json file to object with necessary information for later processing
+        const objTree = {};
+        const keys = Object.keys(obj['data']);
+        for (var repo of keys) {
+            let licenseInfo = obj['data'][repo]['licenseInfo'];
+            // Skip if no license info
+            if (licenseInfo == null) {
+                continue;
+            }
+            if (!mostPopularRepositories.some(d => repo == `${d.owner}/${d.name}`)) {
+                continue;
+            }
+            // Use long name if no short name
+            let licenseName = licenseInfo['spdxId'] == null || licenseInfo['spdxId'] == 'NOASSERTION' ? licenseInfo['name'] : licenseInfo['spdxId'];
+            // Skip if no license name
+            if (licenseName == null) {
+                continue;
+            }
+            if (objTree.hasOwnProperty(licenseName)) {
+                objTree[licenseName][repo] = repo;
+            } else {
+                objTree[licenseName] = {};
+                objTree[licenseName][repo] = repo;
+            }
+        }
+
+        // Converts object tree into d3.hierarchy complient form
```

Comment:
```suggestion
        // Converts object tree into d3.hierarchy compliant form
```
Same typo also in [`js/explore/sunburst_licenses.js`, L305](https://github.com/LLNL/llnl.github.io/blob/c72139812740b49d7073afc88ff9d373daaf9cb2/js/explore/sunburst_licenses.js#L305)